code/game/gamemodes/events/powercreeper.dm

```diff
@@ -78,25 +75,20 @@
 		//add power to powernet
 		add_avail(POWER_PER_FRUIT)
 
-		//spread - copypasta from spreading_growth.dm
-		var/chance = MIN_SPREAD_CHANCE + (powernet.avail / 1000) //two powercreeper plants raise chance by 1
-		chance = chance > MAX_SPREAD_CHANCE ? MAX_SPREAD_CHANCE : chance
-		if(prob(chance))
-			sleep(rand(3,5))
-			if(!gcDestroyed)
-				var/list/neighbours = getViableNeighbours()
-				if(neighbours.len)
-					var/turf/target_turf = pick(neighbours)
-					new /obj/structure/cable/powercreeper(get_turf(target_turf), get_dir(src, target_turf))
-
-		//if there is a person caught in the vines, burn em a bit
-		//electrocute people who aren't insulated
-		if(prob(ATTACK_CHANCE))
-			flick("attacking[add_state]",src)
-			for(var/mob/living/M in range(1, get_turf(src)))
-				spawn
-					Beam(M, "lightning", 'icons/obj/zap.dmi', 5, 2)
-				try_electrocution(M)
+		if(growdirs)
+			var/grow_chance = Clamp(MIN_SPREAD_CHANCE + (powernet.avail / 1000), MIN_SPREAD_CHANCE, MAX_SPREAD_CHANCE)
+			if(prob(grow_chance))
+				var/chosen_dir = pick(cardinal)
+				if(growdirs & chosen_dir)
+					var/turf/target_turf = get_step(src, chosen_dir)
+					if(isViableGrow(target_turf))
+						new /obj/structure/cable/powercreeper(target_turf, get_dir(src, target_turf))
+
+/obj/structure/cable/powercreeper/Crossed(atom/movable/mover, turf/target, height = 1.5, air_group = 0)
```

Comment:
also a major nerf to the volatility of the creeper.
i intended the creeper to be a volatile plant lashing out with bolts of power, you are making it a domesticated pussy

Reply:
Okay. I can revert this

Reply:
i guess you could disable checking if all dirs (even diagonal ones) occupied by creepers or dense

and enable it again using the same way you are fixing `updateNeighbours`

Reply:
Kudzu does this but it's horrible. Still might wanna look at it.

Reply:
well if i actually manage to finish my net datums pr, this should be handled then.

IF

Reply:
Sorted.